src/tree/SubscriptionTreeItem.ts

```diff
@@ -48,7 +48,19 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
     public async createChildImpl(context: ICreateChildImplContext): Promise<AzExtTreeItem> {
         const wizardContext: IStaticWebAppWizardContext = { accessToken: await getGitHubAccessToken(), client: createAzureClient(this.root, WebSiteManagementClient), ...context, ...this.root };
         const title: string = localize('createStaticApp', 'Create Static Web App');
```

Comment:
Add this property so that advanced creation shows up during the tree item picker. https://github.com/microsoft/vscode-azurestorage/blob/main/src/tree/SubscriptionTreeItem.ts#L23